src/core/agent/services/multiSigService.ts

```diff
@@ -1093,6 +1096,219 @@ class MultiSigService extends AgentService {
 
     return { op, exnSaid: exn.ked.d };
   }
+
+  async offerPresentMultisigACDC(
+    multisigSignifyName: string,
```

Comment:
#708 will change this to just be the ID and not signify name - reminder as that will merge first